homeassistant/components/sensorpush_cloud/sensor.py

```diff
@@ -0,0 +1,180 @@
+"""Support for SensorPush Cloud sensors."""
+
+from __future__ import annotations
+
+import itertools
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import (
+    ATTR_DEVICE_ID,
+    ATTR_MODEL,
+    ATTR_NAME,
+    ATTR_TEMPERATURE,
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    UnitOfElectricPotential,
+    UnitOfLength,
+    UnitOfPressure,
+    UnitOfTemperature,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import CONNECTION_BLUETOOTH, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+from homeassistant.util import dt as dt_util
+
+from .const import (
+    ATTR_ALTITUDE,
+    ATTR_ATMOSPHERIC_PRESSURE,
+    ATTR_BATTERY_VOLTAGE,
+    ATTR_BLUETOOTH,
+    ATTR_DEWPOINT,
+    ATTR_HUMIDITY,
+    ATTR_LAST_UPDATE,
+    ATTR_SIGNAL_STRENGTH,
+    ATTR_VAPOR_PRESSURE,
+    DOMAIN,
+    LOGGER,
+    MANUFACTURER,
+    MAX_TIME_BETWEEN_UPDATES,
+)
+from .coordinator import SensorPushCloudConfigEntry, SensorPushCloudCoordinator
+
+SENSORS: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key=ATTR_HUMIDITY,
+        device_class=SensorDeviceClass.HUMIDITY,
+        translation_key="humidity",
```

Comment:
These are not required if I recall correctly, the device class has a default translation so if the value is 'native enough' you can use the default one.

```suggestion
```